.github/workflows/create-release-pr.yml

```diff
@@ -18,4 +19,4 @@ jobs:
         run: yarn run release
         env:
           COMMIT_BODY_HEADER: ":robot: This PR is created automatically ([see releasing](../blob/develop/RELEASING.md)) \n\n"
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.MACHINE_USER_PAT }}
```

Comment:
What user is this for?

Reply:
This one https://github.com/axe-core

I don't think it's named great, but we are using it for other actions and I created a test pr here https://github.com/dequelabs/cauldron/pull/2083

Reply:
Should we add a comment for what user it's for, or do you think it's fine as is?